tests/perf-regression.100threads.30M-keys-i3.yaml

```diff
@@ -17,7 +17,7 @@ append_scylla_args: '--blocked-reactor-notify-ms 4'
 
 # email definitions to send regression report
 send_email: true
-email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com']
+email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com', 'qa@scylladb.com']
```

Comment:
should be `email_recipients: ['qa@scylladb.com']`

Reply:
all of the mentioned persons are part of the QA group

Reply:
I figured that maybe that could ruin some rules the other recipients already have, but sure